packages/fxa-settings/src/components/Settings/PageSecondaryEmailAdd/en.ftl

```diff
@@ -8,3 +8,4 @@ add-secondary-email-enter-address =
   .label = Enter email address
 add-secondary-email-cancel-button = Cancel
 add-secondary-email-save-button = Save
+add-secondary-email-mask = Email masks can’t be used as a secondary email
```

Comment:
Can we leave a comment here that explains what "Email masks" are here? (Localizers for Mozilla Accounts may not be familiar with Firefox Relay.)